### PR TITLE
[Place Page Chart] Annual emissions of greenhouse gas

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: static/js/i18n/strings/en/stats_var_labels.json [hunk 1]
```diff
@@ -1086,5 +1086,9 @@
   "PalmerDroughtSeverityIndex_Atmosphere": {
     "defaultMessage": "PDSI",
     "description": "Label used to describe statistical variables in chart legends, string should be short. Describes the Palmer drought severity index"
+  },
+  "Annual_Emissions_GreenhouseGas_NonBiogenic": {
+    "defaultMessage": "Greenhouse Gas",
+    "description": "Label used to describe statistical variables in chart legends, string should be short. Describes the annual emission of non biogenic greenhouse gas"
```
Comment (pradh) [inline static/js/i18n/strings/en/stats_var_labels.json:1092]: nit: non-biogenic

**Review by pradh — APPROVED** [review]
Nice!

File: static/js/i18n/strings/en/stats_var_labels.json (hunk 1 above)
Comment (shifucun) [reply static/js/i18n/strings/en/stats_var_labels.json]: Done

**Comment by shifucun** [comment]
Thanks for review!

File: server/chart_config/climate.json [hunk 2]
```diff
@@ -78,5 +78,17 @@
       "scale": false
     },
     "isChoropleth": true
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Annual_emissions_greenhouse_gas",
+    "title": "Annual greenhouse gas emissions",
```
Comment (beets) [inline server/chart_config/climate.json:85]: We should clarify that these are large facility emissions:

"Annual greenhouse gas emissions from large facilities"

File: static/js/i18n/strings/en/stats_var_labels.json [hunk 3]
```diff
@@ -1086,5 +1086,9 @@
   "PalmerDroughtSeverityIndex_Atmosphere": {
     "defaultMessage": "PDSI",
     "description": "Label used to describe statistical variables in chart legends, string should be short. Describes the Palmer drought severity index"
+  },
+  "Annual_Emissions_GreenhouseGas_NonBiogenic": {
+    "defaultMessage": "Greenhouse Gas",
```
Comment (beets) [inline static/js/i18n/strings/en/stats_var_labels.json:1091]: Maybe: "Greenhouse Gas (Non-biogenic)"

**Review by beets — COMMENTED** [review]
thanks! just a couple of small wording tweaks

File: server/chart_config/climate.json (hunk 2 above)
Comment (shifucun) [reply server/chart_config/climate.json:85]: Fixed in https://github.com/datacommonsorg/website/pull/1157

**Review by shifucun — COMMENTED** [review]
Thanks for review!

**Pull request merged**